python/felis/datamodel.py

```diff
@@ -391,7 +393,7 @@ class Table(BaseObject):
     primary_key: str | list[str] | None = Field(None, alias="primaryKey")
     """The primary key of the table."""
 
-    tap_table_index: int | None = Field(None, alias="tap:table_index")
+    tap_table_index: int = Field(0, alias="tap:table_index")
```

Comment:
Please retain the ability for the model to distinguish between "missing" and zero `table_index` values.  We may wish to make the handling more sophisticated in the (near) future.

Reply:
I think column_index still allows `None` anyway, right?